pkg/appmanager/appManager.go

```diff
@@ -753,6 +760,7 @@ func (appMgr *Manager) processNextVirtualServer() bool {
 
 	err := appMgr.syncVirtualServer(key.(serviceQueueKey))
 	if err == nil {
+		appMgr.processedItems++
```

Comment:
Do we want to continue this forever (if ints are 64-bit we are okay, if they are 32-bit can't we eventually overflow?